_robots/2023Robot.md

```diff
@@ -0,0 +1,44 @@
+---
+title: 2023 Robot
+subtitle: FlipSide 
+description: Energize Robot 2023
+robot_code: 2023
+layout: robot
+imageicon: "../images/robots/2023robot.png"
+image: "../../images/robots/2023robot.png"
+CompetitionLogo : "../../images/robots/Competition Logos/2023_Charged_Up.svg"
+features:
+    - label: Ranked 1st in the State and 27th in the world
+      icon: fa-certificate
+    - label: Won every attended regional as captain of Alliance 1
+      icon: fa-robot
+    - label: Invited to IRI and played on Alliance 2
+      icon: fa-crosshairs
```

Comment:
These could use some re-wording to keep it more on the GP side or find other items to put here.

Reply:
Better? 
https://github.com/FRCTeam1987/teambroncobots.com/pull/50/commits/f23594b3fcb32a5f7637a99288e87e3d1dd50c12

Reply:
- Try to avoid comparing to other teams / robots - so replace the ranking blurb with something about the robot / team.
- Say something along a 2 time regional winner like the 2022 page.  Undefeated playoffs is a bit much - users interested in performance can go see the linked matches.
- IRI is mentioned in the events section, doesn't need to be mentioned here.